include/PR/controller.h

```diff
@@ -1,14 +1,14 @@
-#ifndef _CONTROLLER_H_
-#define _CONTROLLER_H_
+#ifndef ULTRA64_CONTROLLER_H
+#define ULTRA64_CONTROLLER_H
```

Comment:
```suggestion
#ifndef PR_CONTROLLER_H
#define PR_CONTROLLER_H
```
Should the header guards reflect the change from ultra64 to PR?